### ISSUE TITLE
custom keyPrefix

### ISSUE THREAD

**Issue by aHerbots** [rjsf-team/react-jsonschema-form#1490]
### Prerequisites

- [ x] I have read the [documentation](https://react-jsonschema-form.readthedocs.io/);
- [ ] In the case of a bug report, I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [ ] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net/n1k0/f2y3fq7L/6/) or a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### Description

I'm generating multiple forms on the same page. Some of these Forms have the same properties which causes 'duplicate key' errors
 

#### Expected behavior

It would be really handy if we could pas a keyprefix to prevent this.

#### Actual behavior

'duplicate key' errors

### Version

1.6.5.

**Comment by epicfaace** [comment]
Good point. Maybe `idPrefix` can be prepended to all keys (https://react-jsonschema-form.readthedocs.io/en/latest/advanced-customization/#id-prefix) (as I don't think there would ever be a use case with a need for idPrefix to be defined but not keyPrefix)?

**Comment by aHerbots** [comment]
Yes that would definitely fix my issue since i'm already using the `idPrefix`. I can't see a use case where you would need to define a keyPrefix different than the idPrefix

**Comment by aHerbots** [comment]
a completely different question, do you support constant values? I cant find anything in the documentation about this. Something like if the schema has a const value that the input field is disabled?

**Comment by shvekyha** [comment]
Hi
I can work on this as my first issue:) 
Can you point me to where should this change take place?
Thanks!

**Comment by Jasbir96** [comment]
Hi ,
@epicfaace  i want to contribute to this issue as .
I will be more then happy contribute

**Comment by epicfaace** [comment]
Please do!

**Comment by baristaGeek** [comment]
Is this still an issue?

I can see that in https://github.com/rjsf-team/react-jsonschema-form/pull/1691 the following was added to the docs:

To avoid collisions with existing ids in the DOM, it is possible to change the prefix used for ids (the default is `root`).

**Comment by heath-freenome** [comment]
> Is this still an issue?
> 
> I can see that in [#1691](https://github.com/rjsf-team/react-jsonschema-form/pull/1691) the following was added to the docs:
> 
> To avoid collisions with existing ids in the DOM, it is possible to change the prefix used for ids (the default is `root`).

Closing per this comment
